src/test/kotlin/com/gamingbacklog/api/gamingbacklogapi/unit/services/LibraryServiceTests.kt

```diff
@@ -49,27 +49,33 @@ class LibraryServiceTests {
     mockLibraryDb.clear()
   }
 
+  fun saveMockLibraryDB(libraryId: String, libraryName: String, gameIds: ArrayList<String>) {
+    mockLibraryDb[libraryId] = Library(libraryId, libraryName, gameIds)
+    mockGetOne(libraryId)
+    mockLibraryDb[libraryId]?.let { mockSave(it) }
+  }
+
   @Nested
   @DisplayName("Tests for createLibrary")
   inner class CreateLibrary {
-    @Test
-    fun shouldCreateLibraryWithNoGames() {
-      val library = Library(libraryId, "Backlog", arrayListOf())
+
+    private fun testCreateLibraryUtil(libraryId: String, libraryName: String, gameIds: ArrayList<String>) {
```

Comment:
[Nit] Do we need the "test" prefix? That tends to make me think of an actual test function as opposed to a helper